### DISCUSSION TITLE
Different fiber orientation

### DISCUSSION THREAD

**Discussion by Nikhil23121998** [peridigm/peridigm#301] [Q&A]
Is it possible to model composite with different fiber orientation with the help of coreform cubit?

**Comment by oldninja** [comment]
Could you expand how you plan to do so in Peridigm? 
For a simple geometry, the mesh can be done anywhere, gmsh or even Python. However, the implementation in Peridigm will be the key.

**Reply by Nikhil23121998** [reply]
By applying bond filter I want to see how crack will propagate for different orientation of fibers.
For changing the fiber orientation I need to change orientation of mesh, is it correct?
